### PR TITLE
net-vpn/tor: verify-sig integration

### PR DESCRIPTION
**Pull request by jpds**

### CODE REVIEW

File: app-crypt/openpgp-keys-tor/openpgp-keys-tor-20210425.ebuild [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+DESCRIPTION="OpenPGP keys used by the Tor project"
+HOMEPAGE="https://www.torproject.org/"
+SRC_URI="https://keys.openpgp.org/vks/v1/by-fingerprint/EF6E286DDA85EA2A4BA7DE684E2C6E8793298290 -> EF6E286DDA85EA2A4BA7DE684E2C6E8793298290.asc
+	 https://keys.openpgp.org/vks/v1/by-fingerprint/7A02B3521DC75C542BA015456AFEE6D49E92B601 -> 7A02B3521DC75C542BA015456AFEE6D49E92B601.asc"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86"
+
+S=${WORKDIR}
```
Comment (telans) [inline app-crypt/openpgp-keys-tor/openpgp-keys-tor-20210425.ebuild:15]:
```suggestion
S="${WORKDIR}"
```

File: net-vpn/tor/tor-0.4.4.8.ebuild [hunk 2]
```diff
@@ -3,21 +3,24 @@
...
-	https://archive.torproject.org/tor-package-archive/${MY_PF}.tar.gz"
+	https://archive.torproject.org/tor-package-archive/${MY_PF}.tar.gz
+	verify-sig? ( https://dist.torproject.org/${MY_PF}.tar.gz.asc )"
 S="${WORKDIR}/${MY_PF}"
 
 LICENSE="BSD GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~ppc64 ~x86 ~ppc-macos"
 IUSE="caps doc libressl lzma +man scrypt seccomp selinux +server systemd tor-hardening test zstd"
+VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/torproject.org.asc
```
Comment (telans) [inline net-vpn/tor/tor-0.4.4.8.ebuild:21]: eprefix? Variable placement might be better above description / around the inherits like is seen commonly with python variables

**Review by mgorny — APPROVED** [review]
Technically LGTM but @blueness should approve.

File: net-vpn/tor/tor-0.4.4.8.ebuild (hunk 2 above)
Comment (thesamesam) [reply net-vpn/tor/tor-0.4.4.8.ebuild]: I need to speak to mgorny and patch it to ideally just BROOT it always in the eclass

**Comment by juippis** [comment]
I guess this got caught with libressl removal - @jpds could you please rebase? 

@mgorny did you catch blueness?

**Comment by mgorny** [comment]
> @mgorny did you catch blueness?

Was I supposed to? He's pinged here.

**Comment by jpds** [comment]
@juippis Done (though the QA failure seems unrelated).

**Comment by blueness** [comment]
I'll commit this in a sec.

**Review by blueness — APPROVED** [review]
Looks okay to me.

**Comment by blueness** [comment]
Okay thanks eveyone!  We should be all set.

**Pull request closed without merging**